core/client/src/main/java/alluxio/client/file/FileSystemUtils.java

```diff
@@ -156,7 +156,9 @@ public Boolean apply(Void input) {
         try {
           return fs.getStatus(uri).isPersisted();
         } catch (Exception e) {
-          throw Throwables.propagate(e);
+          Throwables.propagateIfPossible(e);
+          throw new RuntimeException(e);
+          //throw Throwables.propagate(e);
```

Comment:
I see. That's because `propagateIfPossible` is only available after [20.0](https://google.github.io/guava/releases/snapshot/api/docs/com/google/common/base/Throwables.html#throwIfUnchecked-java.lang.Throwable-). However Alluxio still depends on version 14.0, to be [compatible](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/dependency-analysis.html) with Hadoop. So I think this change will have to wait until Hadoop bumps their Guava version